examples/triton-docker-combined/variables.tf

```diff
@@ -0,0 +1,8 @@
+variable "account" {}
+variable "triton_url" {}
+variable "key_path" {}
+variable "key_id" {}
+variable "cert_path" {}
```

Comment:
The env variable is $DOCKER_CERT_PATH so it seems reasonable to add that.

Why not both? Triton_key_path seems okay too.